scraper.py

```diff
@@ -59,3 +73,19 @@
     nhif_inpatient_result = nhif_inpatient_scraper.run_scraper()
     nhif_outpatient_result = nhif_outpatient_scraper.run_scraper()
     nhif_outpatient_cs_result = nhif_outpatient_cs_scraper.run_scraper()
+
+
+if __name__ == "__main__":
+    import multiprocessing
+    # Start the scrapers
+    scraping = multiprocessing.Process(target=scrapers)
+    scraping.start()
+    scraping.join(30*60)
+
+    # log error if scraping is still running after 30 minutes
+    if scraping.is_alive():
+        # create a random Id for this scrap instance
+        import random
+        scraper_id = random.randint(1, 100000)
+        error['MESSAGE'] = 'Scraper: {} is taking more than 30 minutes'.format(scraper_id)
+        scraper.print_error(error)
```

Comment:
Because it isn't exactly an error, let's have this as a warning instead.

Reply:
Yea, I have change the log level to warning.